common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -39,6 +39,22 @@
      */
     public static final Charset ENCODING_UTF8 = Charset.forName(ENCODING_UTF8_STRING);
 
+    /**
+     * The Constant ASCII.
+     */
+    public static final String ENCODING_ASCII_STRING = "ASCII";
+
+    /**
+     * The Constant CHARSET_ASCII.
+     */
+    public static final Charset CHARSET_ASCII = Charset.forName(ENCODING_ASCII_STRING);
+
+    /**
+     * A constant for PMD to be happy with.
```

Comment:
Can we elaborate on this?

Reply:
I think it's just there to suppress PMD warning.